projects/plugins/social/src/js/store/actions/jetpack-settings.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+/*eslint lodash/import-scope: [2, "method"]*/
+import pick from 'lodash/pick';
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import {
+	fetchJetpackSettings,
```

Comment:
Are we calling it Jetpack Settings because the store would be for all Jetpack settings? I wonder if it should be Social Settings or something, but that's a fairly major change!

Reply:
It's named that way because it handles Jetpack-related settings (in this case the module status). I'm far from an expert when it comes to stores though so I'm very open to suggestions.

Reply:
Yeah I was thinking that, but it seemed quite specific to Social/publicize, and I thought maybe we should make the naming more specific too.